Makefile

```diff
@@ -30,6 +34,10 @@ wire-gen:
 	echo "Accessing directory and wire all DI $(WIRE_DIR)/$$dir"; \
 	cd $(WIRE_DIR)/$$dir && wire
 
+# Generate OpenAPI Docs
+docs-gen:
+	swag fmt && swag init -g $(OPENAPI_ENTRY_POINT) -o $(OPENAPI_OUTPUT_DIR) -ot $(OPENAPI_OUTPUT_TYPE)
+
```

Comment:
tambahkan di README how to use nya @KennFatt

Reply:
```
docs-gen:
```

convention gw yg baru, ga pake suffix `-gen` 
langsung aja `make docs`

Reply:
mas @adityasatrio punten, gabisa make `make docs` mas soalny ada directory namanya `docs` juga jd conflict. Gw rename jadi `make swagger` ya